packages/types/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "declaration": true,
+    "outDir": "dist",
+    "strict": true,
+    "moduleResolution": "node",
+    "target": "ES6",
+    "module": "CommonJS"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "node_modules"]
+}
```

```suggestion
{
  "extends": "@repo/typescript-config/base.json",
  "include": ["src/**/*"],
  "exclude": ["dist", "build", "node_modules"]
}
```
모노레포답게 base.json에서 끌어사용한건 어때?